guardrails/src/drivers/rest/routers/admin.py

```
@@ -0,0 +1,273 @@
+
+    logger.info(f"Activated version {version_id} with revision {version.revision}")
+
+    return ActivateResponse(
+        status="activated",
+        version_id=version.id,
+        revision=version.revision,
+    )
+
+
+@router.post("/{domain_id}/reload", response_model=ReloadResponse)
+async def reload_runtime(domain_id: UUID) -> ReloadResponse:
```

Ruff (0.14.8)

Unused function argument: `domain_id`

(ARG001)